corehq/warehouse/models.py

```diff
@@ -62,3 +62,88 @@ def fail(self):
     )
     def decommission(self):
         return
+
+
+class BaseDim(models.Model):
+    domain = models.CharField(max_length=255)
```

Comment:
i like the idea of always including the domain with each dim to ensure we never mix up data and make it easier to reason about. however it's probably not entirely necessary, let me know if you think we should remove